src/pages/OfferNew/Checkout/index.tsx

```diff
@@ -209,7 +206,7 @@ export const Checkout: React.FC<Props> = ({
       offerData.person.ssn,
   )
 
-  if (signStatus?.signState === SignState.Completed) {
+  if (hasContract) {
```

Comment:
We could also do this to make it super-safe
```suggestion
  if (signStatus?.signState === SignState.Completed && hasContract) {
```

Reply:
Where we check both the first and last step of the sign process. Right now creating the contract is last in the chain so to speak.